.eslintrc.js

```diff
@@ -284,6 +284,7 @@ module.exports = {
                 patterns: [ "@wso2is/**/dist/**", "lodash/**", "lodash/fp/**" ]
             }
         ],
+        "no-trailing-spaces": "warn",
```

Comment:
Is there a `.editorconfig` rule for this as well?

Reply:
`trim_trailing_whitespace` this config is there. Already added to `. editorconfig`